resources/assets/js/app/lang/global.locale.json

```diff
@@ -101,5 +101,12 @@
         "<p><strong>BeBeep</strong> — Ваш надійний помічник у пошуку спільних подорожей. Ми об’єднуємо водіїв і пасажирів, яким по дорозі, для міжміських подорожей за дешевшими цінами.</p><p>З <strong>BeBeep</strong> Ваша наступна подорож буде вигідною та зручною!</p>",
         "<p><strong>BeBeep</strong> — Ваш надежный помощник в поиске общих поездок. Мы объединяем водителей и пассажиров, которым по дороге, для междугородных поездок по более дешевым ценам.</p><p>С <strong>BeBeep</strong> Ваша следующая поездка будет выгодной и удобной!</p>"
     ]
+  },
+  "errors": {
+    "5xx_internal_server": [
+      "Internal Server Error. Please, contact the server administrator.",
```

Comment:
no need to give user extended info. Will be enough message like "Something went wrong. Please try later"